client/task-list/index.js

```diff
@@ -302,9 +302,14 @@ class TaskDashboard extends Component {
 					{ task.completed && <Icon icon={ check } /> }
 				</div>
 			);
-			task.after = ( task.time
-				? <span className="woocommerce-task-estimated-time">{ task.time }</span>
-				: <Icon icon={ chevronRight } />
+			task.after = task.time ? (
+				! task.completed && (
+					<span className="woocommerce-task-estimated-time">
+						{ task.time }
+					</span>
+				)
+			) : (
```

Comment:
Are the arrows supposed to show on completed tasks or only when the entire task list is completed?  If the former, we should update this to:

```suggestion
			task.after = task.time && ! task.completed ? (
				<span className="woocommerce-task-estimated-time">
					{ task.time }
				</span>
			) : (
```

Current for reference:
<img width="531" alt="Screen Shot 2020-05-25 at 11 18 28 AM" src="https://user-images.githubusercontent.com/10561050/82794210-c5cd8280-9e7a-11ea-9d98-0052fde9b4c6.png">


And with the above suggestion:
<img width="466" alt="Screen Shot 2020-05-25 at 11 27 20 AM" src="https://user-images.githubusercontent.com/10561050/82794219-c82fdc80-9e7a-11ea-94a5-607edd960de9.png">

Reply:
It looks like the chevron is just a fallback for when there isn't a time estimate for the task.

The designs don't show a chevron on completed tasks:

![Screen Shot 2020-05-25 at 9 15 29 AM](https://user-images.githubusercontent.com/63922/82825754-04f2d800-9e69-11ea-9ffe-25670f3bc558.png)


(Which means this will need an update to not show the chevron on completed tasks that don't have a time estimate)